install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -212,6 +212,14 @@ rules:
       - list
       - patch
 
+  - apiGroups:
```

Comment:
why do we need this, the role already has `machine.openshift.io` for `openshift-machine-api`?

Reply:
This is what we get if we dont add these roles:

```
E0710 14:33:47.569176       1 reflector.go:126] github.com/openshift/cluster-api/pkg/client/informers_generated/externalversions/factory.go:118: Failed to list *v1beta1.MachineSet: machinesets.machine.openshift.io is forbidden: User "system:serviceaccount:openshift-machine-api:machine-api-operator" cannot list resource "machinesets" in API group "machine.openshift.io" at the cluster scope
```

Reply:
machine-api-operator cluster role does not have permission to list/watch resources that belong to machine.openshift.io group. Thats what my understanding is.

Reply:
that's because you are telling the informer to watch cluster wide. Can you please narrow the scope to the passed namespace? i.e same that we are doing for the kube informer `informers.WithNamespace(targetNamespace)`

Reply:
clusterrole `machine-api-operator` has permissions only  for `config.openshift.io` api group. Even after namespaced informers, i am getting following error:
```
Failed to list *v1beta1.Machine: machines.machine.openshift.io is forbidden: User "system:serviceaccount:openshift-machine-api:machine-api-operator" cannot list resource "machines" in API group "machine.openshift.io" in the namespace "openshift-machine-api"
``` 
I think we will need to add permissions that i had initially.